src/app/badges/page.tsx

```diff
@@ -301,6 +301,12 @@ export default function Badges() {
               anchor={anchor}
               open={drawerStates.walletOperations[anchor]}
               onClose={() => handleToggleDrawer(anchor)}
+              PaperProps={{
+                style: {
+                  width: '390px',
+                  left: 'calc(50% - 195px)', // 50% - half of width
```

Comment:
Given our priorities for speed atm I am ok with this cc @siwan-cb

Reply:
also this was the only method that worked, using flexbox, mounting to a different container other than document.body both yielded super broken results